src/main/webapp/i18n/de/instructorCourseDashboard.json

```diff
@@ -10,11 +10,14 @@
             "tutors": "Tutoren",
             "assessments": "Assessments",
             "open": "offen",
+            "openComplaint": "offen Beshwerden",
+            "openRequest": "offen Anfragen",
             "noExercise": "Für diesen Kurs gibt es keine Übung.",
             "exercise": "Übung",
             "dueDate": "Abgabedatum",
             "assessmentDueDate": "Assessment Abgabedatum",
             "complaints": "Beschwerden",
+            "moreFeedbackRequests": "Mehr Feedback Anfragen",
```

Comment:
horrible translation! Please fix it!

Reply:
How should I fix it? I am not sure what is a better translation here.